user_manager/api/v1/views.py

```diff
@@ -0,0 +1,419 @@
+
+            If the request for information about the managers is successful, an HTTP 200 "OK"
+            response is returned with a collection of managers.
+
+            The HTTP 200 response has the following values.
+
+            * count: The number of managers in a course.
+
+            * next: The URI to the next page of results.
+
+            * previous: The URI to the previous page of results.
+
+            * num_pages: The number of pages.
+
+            * results: a list of manager users:
+
+                * id: The user id for a manager user, or null if manager doesn't have an
+                    account yet.
+
+                * email: Email address of manager.
+
+        **Example GET Response**
+
+            {
+                "count": 99,
+                "next": "https://courses.org/api/user_manager/v1/managers/?page=2",
+                "previous": null,
+                "results": {
+                    {
+                        "email": "staff@example.com",
+                        "id": 9
+                    },
+                    { ... }
+                }
+            }
+    """
+
+    serializer_class = ManagerListSerializer
+    queryset = UserManagerRole.objects.values(
+        'manager_user',
+        'manager_user__email',
+        'unregistered_manager_email',
+    ).distinct()
+
+
+class ManagerReportsListView(ManagerViewMixin, ListCreateAPIView):
+    """
+        **Use Case**
+
+            * Get a list of all users that are reports for the provided manager.
+
+            * Add a user as a report under a manger.
+
+            * Remove a user or all users under a manager.
+
+        **Example Request**
+
+            GET /api/user_manager/v1/reports/{user_id}/
+
+            POST /api/user_manager/v1/reports/{user_id}/ {
+                "email": "{email}"
+            }
+
+            DELETE /api/user_manager/v1/reports/{user_id}/
+
+            DELETE /api/user_manager/v1/reports/{user_id}/?user={user_id}
+
+        **GET Parameters**
+
+            * user_id: username or email address for user whose reports you want fetch
+
+        **POST Parameters**
+
+            * user_id: username or email address for user for whom you want to add a manger
+
+            * email: Email address for a user
+
+        **DELETE Parameters**
+
+            * user_id: username or email address for user
+
+        **GET Response Values**
+
+            If the request for information about the managers is successful, an HTTP 200 "OK"
+            response is returned with a collection of managers.
+
+            The HTTP 200 response has the following values.
+
+            * count: The number of managers in a course.
+
+            * next: The URI to the next page of results.
+
+            * previous: The URI to the previous page of results.
+
+            * num_pages: The number of pages.
+
+            * results: a list of users under a manager:
+
+                * id: The user id for a user.
```

Comment:
Same here - this should be a username, not a user ID. Especially since this API accepts usernames, not IDs, as parameters.